File: GZCTF/ClientApp/src/client/models/AccountVerifyModel.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

/**
 * 账号验证
 */
export type AccountVerifyModel = {
    /**
     * 邮箱接收到的Base64格式Token
     */
    token: string;
    /**
     * 用户邮箱的Base64格式
     */
    email: string;
};

File: src/GZCTF/Models/Request/Account/AccountVerifyModel.cs
using System.ComponentModel.DataAnnotations;

namespace GZCTF.Models.Request.Account;

/// <summary>
/// Account verification
/// </summary>
public class AccountVerifyModel
{
    /// <summary>
    /// Base64 formatted token received via email
    /// </summary>
    [Required(ErrorMessageResourceName = nameof(Resources.Program.Model_TokenRequired),
        ErrorMessageResourceType = typeof(Resources.Program))]
    public string? Token { get; set; }

    /// <summary>
    /// Base64 formatted user email
    /// </summary>
    [Required(ErrorMessageResourceName = nameof(Resources.Program.Model_EmailRequired),
        ErrorMessageResourceType = typeof(Resources.Program))]
    public string? Email { get; set; }
}
